doc/nrf/releases_and_maturity/releases/release-notes-3.2.0.rst

```diff
@@ -0,0 +1,1352 @@
+  * The :ref:`nrf_cloud_coap_fota_sample` sample to demonstrate how to use the `nRF Cloud CoAP API`_ for FOTA updates.
+  * The :ref:`nrf_cloud_coap_device_message` sample to demonstrate how to use the `nRF Cloud CoAP API`_ for device messages.
+  * The :ref:`nrf_cloud_mqtt_device_message` sample to demonstrate how to use the `nRF Cloud MQTT API`_ for device messages.
+  * The :ref:`nrf_cloud_mqtt_fota` sample to demonstrate how to use the `nRF Cloud MQTT API`_ for FOTA updates.
+  * The :ref:`nrf_cloud_mqtt_cell_location` sample to demonstrate how to use the `nRF Cloud MQTT API`_ for nRF Cloud's cellular location service.
+
+* Updated the following samples to use the new ``SEC_TAG_TLS_INVALID`` definition:
+
+  * :ref:`modem_shell_application`
+  * :ref:`http_application_update_sample`
+  * :ref:`http_modem_delta_update_sample`
+  * :ref:`http_modem_full_update_sample`
+
+* Removed:
+
+  * The SLM Shell sample.
+    Use the sample from `Serial Modem`_ instead.
+  * The deprecated LTE Sensor Gateway sample.
+
+* :ref:`nrf_cloud_rest_cell_location` sample:
+
+  * Added runtime setting of the log level for the nRF Cloud logging feature.
+
+* :ref:`modem_shell_application` sample:
+
+  * Added:
+
+    * Support for environment evaluation using the ``link enveval`` command.
+    * Support for NTN NB-IoT to the ``link sysmode`` and ``link edrx`` commands.
+    * Support for Non-Terrestrial Network (NTN) helper functionality using the ``ntn`` command.
+
+  * Updated the PDN connection management to use the PDN functionality in the :ref:`lte_lc_readme` library instead of the :ref:`pdn_readme` library.
+
+* :ref:`pdn_sample` sample:
+
+  * Updated the PDN functionality to use the PDN management in the :ref:`lte_lc_readme` library instead of the :ref:`pdn_readme` library.
+
+* :ref:`gnss_sample` sample:
+
+  * Added TLS support for connection to the SUPL server.
+
+* :ref:`nrf_cloud_multi_service` sample:
+
+  * Fixed an issue where sporadically the application was stuck waiting for the device to connect to the internet.
+    This was due to wrong :ref:`Connection Manager <zephyr:conn_mgr_overview>` initialization.
+  * Deprecated the sample.
+
+Cryptography samples
+--------------------
+
+* Added:
+
+  * Support for the ``nrf54lv10dk/nrf54lv10a/cpuapp`` and ``nrf54lv10dk/nrf54lv10a/cpuapp/ns`` board targets to all samples (except :ref:`crypto_test`).
+  * Support for the ``nrf54lm20dk/nrf54lm20a/cpuapp/ns`` board target in all supported cryptography samples.
+  * The :ref:`crypto_kmu_cracen_usage` sample.
+
+* Updated documentation of all samples for style consistency: added lists of cryptographic features used by each sample and sample output in the testing section.
+
+* :ref:`crypto_persistent_key` sample:
+
+  * Added support for the ``nrf54h20dk/nrf54h20/cpuapp`` board target, demonstrating use of Internal Trusted Storage (ITS) on the nRF54H20 DK.
+
+* :ref:`crypto_aes_ctr` sample:
```

Comment:
Could add as following under Added:

  * Support for the ``nrf54lm20dk/nrf54lm20a/cpuapp`` board target in the following samples:
    
    * :ref:`crypto_aes_ctr`
    * :ref:`crypto_aes_cbc`